### PR TITLE
feat: output FPS for go pipeline

### PR DESCRIPTION
**Pull request by vli11**
## PR Checklist

- [x] Added label to the Pull Request for easier discoverability and search
- [x] Commit Message meets guidelines as indicated in the URL https://github.com/edgexfoundry/edgex-go/blob/main/.github/Contributing.md
- [x] Every commit is a single defect fix and does not mix feature addition or changes
- [ ] Unit Tests have been added for new changes
- [ ] Updated Documentation as relevant to the changes
- [x] All commented code has been removed
- [ ] If you've added a dependency, you've ensured license is compatible with repository license and clearly outlined the added dependency.
- [ ] PR change contains code related to security
- [ ] PR introduces changes that breaks compatibility with other modules (If YES, please provide details below)

## What are you changing?
grpc-go pipeline's output result is changing from latency.txt to pipeline#.log and r#.jsonl to align with python's output result

## Issue this PR will close

close: #**issue_number**

## Test Instructions if applicable
- run camera simulator
- make build-grpc-go
- make build-ovms-client
- run this script: PIPELINE_PROFILE="grpc_go" sudo -E ./docker-run.sh --workload opencv-ovms --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0
- you should see the result logs inside results/ directory

To test Debug mode:
- add `-e DEBUG="1" \` after `-e PIPELINE_PROFILE="$PIPELINE_PROFILE" \` in docker-run-opencv-ovms.sh
- make build-grpc-go
-  run this script: PIPELINE_PROFILE="grpc_go" sudo -E ./docker-run.sh --workload opencv-ovms --platform core --inputsrc rtsp://127.0.0.1:8554/camera_0
- you should not see any result logs inside results/ directory, instead, run `docker logs dev`, you should see the results output on the console.

### CODE REVIEW

File: configs/opencv-ovms/grpc_go/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+#
+# Copyright (C) 2023 Intel Corporation.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+./grpc-go -i $inputsrc -u 127.0.0.1:$GRPC_PORT -o /tmp/results 2>&1  | tee >/tmp/results/r$cid_count.jsonl >(stdbuf -oL sed -n -e 's/^.*fps: //p' | stdbuf -oL cut -d , -f 1 > /tmp/results/pipeline$cid_count.log)
```
Comment (jim-wang-intel) [inline configs/opencv-ovms/grpc_go/entrypoint.sh:8]: do we still need -o /tmp/results?

Comment (brian-intel) [inline configs/opencv-ovms/grpc_go/entrypoint.sh]: Can fix in a future PR but the IP should use $HOST which is another flag available in the go program

**Review by brian-intel — APPROVED** [review]

**Review by jim-wang-intel — APPROVED** [review]
LGTM

File: configs/opencv-ovms/grpc_go/main.go [hunk 2]
```diff
@@ -123,19 +108,13 @@ func runModelServer(client *grpc_client.GRPCInferenceServiceClient, webcam *gocv
 		if err != nil {
 			fmt.Printf("post process failed: %v\n", err)
 		}
-		fmt.Printf("length of detectedObjects after  processing: %v\n", len(detectedObjects.Objects))
-		detectedObjects = detectedObjects.FinalPostProcessAdvanced()
```
Comment (antoniomtz) [inline configs/opencv-ovms/grpc_go/main.go:127]: I believe we need this line back

**Review by antoniomtz — CHANGES_REQUESTED** [review]

File: configs/opencv-ovms/grpc_go/main.go (hunk 2 above)
Comment (jim-wang-intel) [reply configs/opencv-ovms/grpc_go/main.go:127]: yea we need `detectedObjects = detectedObjects.FinalPostProcessAdvanced()` line back, that is a good catch!

**Review by antoniomtz — APPROVED** [review]
LGTM

**Pull request merged**